furusiya/function_mess.py

```diff
@@ -15,8 +10,11 @@
 from model.rect import Rect
 from model.tile import Tile
 from model.weapons import Bow
-from constants import *
-
+from model.monsters import monster_factory
+from random import randint
+import random
+import shelve
+import tdl
```

Comment:
Slight nitpick: The pep8 convention calls for organizing imports in the following way:
`default packages`
`\n`
`third party packages`
`\n`
`your own packages`

Reply:
I did not know that. InshaAllah will fix. I thought order is supposed to be alphabetical.